plugins/inputs/ceph/ceph.go

```diff
@@ -411,26 +432,53 @@ func decodeStatus(acc telegraf.Accumulator, input string) error {
 	return nil
 }
 
+// decodeStatusFsmap decodes the FS map portion of the output of 'ceph -s'
+func decodeStatusFsmap(acc telegraf.Accumulator, data *CephStatus) error {
+	fields := map[string]interface{}{
+		"up":         data.FSMap.NumUp,
+		"in":         data.FSMap.NumIn,
+		"max":        data.FSMap.NumMax,
+		"up_standby": data.FSMap.NumUpStandby,
+	}
+	acc.AddFields("ceph_fsmap", fields, map[string]string{})
+	return nil
+}
+
 // decodeStatusHealth decodes the health portion of the output of 'ceph status'
 func decodeStatusHealth(acc telegraf.Accumulator, data *CephStatus) error {
+	statusCodes := map[string]float64{
+		"HEALTH_ERR":  0,
+		"HEALTH_WARN": 1,
+		"HEALTH_OK":   2,
```

Comment:
Is this something commonly used in Ceph context? As usually `0` is the code for `OK`.

If this is not commonly used in Ceph, I would leave it out, as a user that would want this can do this by using the enum processor.

Reply:
I agree that `0` is normally the code for `OK`, but I just copied the values from the Ceph [definitions](https://github.com/ceph/ceph/blob/a67d1cf2a7a4031609a5d37baa01ffdfef80e993/src/include/health.h#L12) on status codes.

I don't know if it's commonly used by people, but my thought process is was to save the enum processor step, and just do it in the plugin. That way both a textual, and numeric representations exist and people can pick the one they want.

Reply:
Okay, you convinced me with the link to Ceph code showing the same. It can stay.